superset_config.py

```diff
@@ -161,7 +161,9 @@ class CeleryConfig(object):
                 "DASHBOARD_NATIVE_FILTERS_SET": True,
                  "GENERIC_CHART_AXES": True,
                 "EMBEDDED_SUPERSET": True,
-                 "ENABLE_TEMPLATE_PROCESSING": True}
+                 "ENABLE_TEMPLATE_PROCESSING": True,
+                 "THUMBNAILS": True,
+                 "THUMBNAIL_CACHE_CONFIG": CacheConfig = { "CACHE_TYPE": "null", "CACHE_NO_NULL_WARNING": True}}
```

Comment:
We probably could cache these somewhere, but not knowing how it's used, I'm fine leaving the cache disabled unless we know it's causing issues.